### PR TITLE
pandoc 1.15.0.4

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

**Comment by DomT4** [comment]
Are the manpages no longer available for installation?

**Comment by adunning** [comment]
According to the release notes, 'Man page is now built and installed as part of the cabal build process.'

**Comment by DomT4** [comment]
Neat, Thanks.

**Comment by mikemcquaid** [comment]
Install failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/28872/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_pandoc/

**Comment by adunning** [comment]
Sorry about that; will get this fixed.

File: Library/Formula/pandoc.rb [hunk 1]
```diff
@@ -5,13 +5,12 @@ class Pandoc < Formula
...
   homepage "http://pandoc.org"
-  url "https://hackage.haskell.org/package/pandoc-1.14.0.4/pandoc-1.14.0.4.tar.gz"
-  sha256 "01955bfb1f397ec22bbce10e2df7b4f2214b7289bf79bf51eb7ae0e3b427fadf"
+  url "https://hackage.haskell.org/package/pandoc-1.15.0.3/pandoc-1.15.0.3.tar.gz"
+  sha256 "e9759e50263b803a40b339534444ff0bd1371dbe299927d113544d746570dcaf"
 
   head "https://github.com/jgm/pandoc.git"
 
   bottle do
-    revision 1
```
Comment (bfontaine) [inline Library/Formula/pandoc.rb]: You can leave this line here, it’ll be updated by our bot.

**Comment by dunn** [comment]
1.15.0.3 can't build in a sandbox; see https://github.com/jgm/pandoc/issues/2260. 1.15.0.4 fixed the issue.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/b0fc17e373db80851ab052e6a425d79a4a371e37. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**